waspc/data/Generator/templates/sdk/wasp/server/_types/index.ts

```diff
@@ -1,41 +1,52 @@
 {{={= =}=}}
-import { type Expand } from 'wasp/universal/types';
+import { type Expand } from 'wasp/universal/types'
 import { type Request, type Response } from 'express'
-import { type ParamsDictionary as ExpressParams, type Query as ExpressQuery } from 'express-serve-static-core'
+import {
+  type ParamsDictionary as ExpressParams,
+  type Query as ExpressQuery,
+} from 'express-serve-static-core'
 import { prisma } from 'wasp/server'
 {=# isAuthEnabled =}
-import {
-  type {= authIdentityEntityName =},
-} from "wasp/entities"
-import {
-  type EmailProviderData,
-  type UsernameProviderData,
-  type OAuthProviderData,
-} from 'wasp/auth/utils'
 import { type AuthUser } from 'wasp/auth'
 {=/ isAuthEnabled =}
-import { type _Entity } from "./taggedEntities"
-import { type Payload } from "./serialization";
+import { type _Entity } from './taggedEntities'
+import { type Payload } from './serialization'
 
 export * from "./taggedEntities"
 export * from "./serialization"
 
-export type Query<Entities extends _Entity[], Input extends Payload, Output extends Payload> = 
-  Operation<Entities, Input, Output>
+export type UnauthenticatedQueryDefinition<
```

Comment:
The `Definition` suffix is appropriate here, as we're using these generics to construct the types users assign to their operation definitions.

We want to use `Query`, `Action`, etc. for typing the user-facing API for calling the operations (see `wrappers.ts`).